source/_includes/integrations/remove_device_service.md

```diff
@@ -1,5 +1,3 @@
 ### To remove an integration instance from Home Assistant
 
```

Comment:
```suggestion
```
The suggestion would be to remove this header and leave the rest as it was, but I'm also curious what ideas Frenck has.